packages/remirror__extension-annotation/__tests__/annotation-extension.spec.ts

```diff
@@ -311,102 +311,149 @@ describe('styling', () => {
 });
 
 describe('helpers', () => {
-  it('#getAnnotations', () => {
-    const {
-      add,
-      helpers,
-      nodes: { p, doc },
-      commands,
-    } = create();
-
-    add(doc(p('<start>Hello<end>')));
-    commands.addAnnotation({ id: '1' });
+  describe('#getAnnotations', () => {
+    it('default', () => {
+      const {
+        add,
+        helpers,
+        nodes: { p, doc },
+        commands,
+      } = create();
+
+      add(doc(p('<start>Hello<end>')));
+      commands.addAnnotation({ id: '1' });
+
+      expect(helpers.getAnnotations()).toEqual([
+        {
+          id: '1',
+          from: 1,
+          to: 6,
+          text: 'Hello',
+        },
+      ]);
+    });
 
-    expect(helpers.getAnnotations()).toEqual([
-      {
-        id: '1',
-        from: 1,
-        to: 6,
-        text: 'Hello',
-      },
-    ]);
-  });
+    it('gracefully handles misplaced annotations', () => {
+      const {
+        add,
+        helpers,
+        nodes: { p, doc },
+        commands,
+      } = create();
+
+      add(doc(p('Hello')));
+      commands.setAnnotations([
+        {
+          id: '1',
+          from: 100,
+          to: 110,
+        },
+      ]);
 
-  it('#getAnnotations gracefully handles misplaced annotations', () => {
-    const {
-      add,
-      helpers,
-      nodes: { p, doc },
-      commands,
-    } = create();
+      expect(helpers.getAnnotations()[0]?.text).toBeUndefined();
```

Comment:
I wouldn't be sure what to expect here. Might be good to make this clearer in the test, e.g. `expect(helpers.getAnnotations()[0].text).toBeUndefined();` or `expect(helpers.getAnnotations().length).toBe(0);`

Reply:
Ah, I haven't changed these tests, just group them within a new `describe` block for simplicity